### DISCUSSION TITLE
<< Extra iformation for URL >>

### DISCUSSION THREAD

**Discussion by icamps** [keepassxreboot/keepassxc#8554] [Q&A] [answered]
Hello,

I am newbie to KeePassXC so I read the documentation but didn't find anything related...

I wonder if it can differentiate two different sites that used the same URL root. 

I have access to scientific journals that used the same system, so, the same URL root: [https://www.editorialmanager.com/](https://www.editorialmanager.com/) plus the journal identification. So, each journal add its own code after the URL, for example: https://www.editorialmanager.com/apsusc/ is for one journal, whereas https://www.editorialmanager.com/jmmo is for another one.

Can KeePassXC understand and separate both sites?

Regards,

Camps

**Comment by droidmonkey** [comment, marked as answer]
Yes we will recognize the difference. You need to enable the browser integration settings of "Only return best matching credentials"
